numba/npyufunc/parfor.py

```diff
@@ -366,16 +367,26 @@ def _lower_parfor_parallel(lowerer, parfor):
                     if isinstance(inst, ir.Assign):
                         rhs = inst.value
                         # We probably need to generalize this since it only does substitutions in
-                        # inplace_ginops.
+                        # inplace_binops.
                         if isinstance(rhs, ir.Expr) and rhs.op == 'inplace_binop' and rhs.rhs.name == init_var.name:
+                            if config.DEBUG_ARRAY_OPT:
+                                print("Adding call to reduction", rhs)
+                            if rhs.fn == operator.isub:
+                                rhs.fn = operator.iadd
+                                rhs.immutable_fn = operator.add
+                            if rhs.fn == operator.itruediv or rhs.fn == operator.ifloordiv:
+                                rhs.fn = operator.imul
+                                rhs.immutable_fn = operator.mul
+                            if config.DEBUG_ARRAY_OPT:
```

Comment:
Do the corresponding docs need an update: https://github.com/numba/numba/blob/5bd018fd55ab2ff0916ab2c6360334fe94e471a4/docs/source/user/parallel.rst (under "Supported Operations") ?

Reply:
+= and *= are not mentioned in Supported Operations.  They are mentioned in the Explicit Parallel Loops section.  I have updated that section to include -= and /=.

Reply:
Great, thanks.